index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--Header-->
+    <div class="wrapper">
+        <header>
+            <a href="#" id="logo">My Logo</a>
+            <nav>
+                <button class="hamburger" id="menu-btn-open" type="button">
+                    <span class="hamburger-top"></span>
+                    <span class="hamburger-middle"></span>
+                    <span class="hamburger-bottom"></span>
+                </button>
+            </nav>
+        </header>
+        <section class="main-content">
+            <div class="main-content-container">
+                <div>
+                    <h1>I'm Mariane <br/>
+                        Glad to see you!
+                    </h1>
+                </div>
+                <div>
+                    <p>
+                        I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded, don't hestiate to contact me.
+                    </p>
+                </div>
+                <div>
+                    <p id="social-heading">Let's Connect</p>
```

Comment:
- [ ] kindly consider adding this social-heading in a header tag `hx (examples: <h2>,<h3>, etc...)` to make it  semantically correct to the screen readers and other technologies will thank you